app/models/concerns/sufia/solr_document_behavior.rb

```diff
@@ -57,6 +57,16 @@ def collection_ids
       Array.wrap(self['collection_ids_tesim'])
     end
 
+    def embargo_release_date
+      date_field_name = Hydra.config.permissions.embargo.release_date.sub(/_dtsi/, '')
```

Comment:
@mjgiarlo hydra-head is already specifying the full solr field - https://github.com/projecthydra/hydra-head/blob/master/hydra-access-controls/lib/hydra/config.rb#L140

So, yeah, a format_field method would be more straight-forward.